controllers/templateupdaterequest/templateupdaterequest_controller_test.go

```diff
@@ -690,6 +692,159 @@ func TestReconcile(t *testing.T) {
 
 	})
 
+	t.Run("spaces", func(t *testing.T) {
+		t.Run("when space tier hash label still the same", func(t *testing.T) {
+			// given
+			previousBasicTier := tiertest.BasicTier(t, tiertest.PreviousBasicTemplates)
+			basicTier := tiertest.BasicTier(t, tiertest.CurrentBasicTemplates, tiertest.WithCurrentUpdateInProgress())
+			space := spacetest.NewSpace("user-1", spacetest.WithTierNameAndHashLabelFor(previousBasicTier)) // space's hash matches TUR hash
+			initObjs := []runtime.Object{basicTier, space}
+			previousHash, err := tierutil.ComputeHashForNSTemplateTier(previousBasicTier)
+			require.NoError(t, err)
+			initObjs = append(initObjs, turtest.NewTemplateUpdateRequest("user-1", *basicTier, turtest.CurrentTierHash(previousHash)))
+			r, req, cl := prepareReconcile(t, initObjs...)
```

Comment:
I guess that we could pass the init objects directly to the `prepareReconcile` function?
```suggestion
			previousHash, err := tierutil.ComputeHashForNSTemplateTier(previousBasicTier)
			require.NoError(t, err)
			r, req, cl := prepareReconcile(t, basicTier, space, turtest.NewTemplateUpdateRequest("user-1", *basicTier, turtest.CurrentTierHash(previousHash)))
```
the same applies for the following cases as well.

Reply:
Done in 634cc5e4be3e3398ad3a3c36e418b20c92b1cee1